src/modules/sched-simple/sched.c

```diff
@@ -621,6 +661,8 @@ int mod_main (flux_t *h, int argc, char **argv)
 
     if (simple_sched_init (h, ss) < 0)
         goto done;
+    /* must be called after schedutil_create() to ensure sched service
+     * registered */
```

Comment:
I don't think there's an actual ordering requirement here?    Maybe the comment should just read
```c
/* N.B. simple_sched_init() calls schedutil_create(),
 * which registers the "sched" service  name
 */
```

Reply:
> I don't think there's an actual ordering requirement here?

oh, can you actually add a handler before the service is registered?  Did not know that.

Reply:
Yep.  Message handler registrations are independent of service registration.  The downside is its possible to register handlers that have no possibility of being called.  (Well, not strictly true: `flux_requeue()` lets you stuff arbitrary messages into the receive queue).